package.json

```diff
@@ -33,7 +33,7 @@
     "eslint-plugin-jsx-a11y": "^2.2.1",
     "eslint-plugin-react": "^6.2.0",
     "glob": "^7.1.1",
-    "gulp": "github:gulpjs/gulp#6d71a65",
+    "gulp": "^4.0.0",
```

Comment:
This change more-or-less reverses Commit fb6208ee, which has the commit message

> Lock gulp to pre-alpha.3 to avoid vinyl break

Is switching to a more recent version of `gulp` going to cause a regression with `vinyl` (whatever that is)?

If we want to restore the "same" version of `gulp` that this project was using, then I think we can use

```json
    "gulp": "github:gulpjs/gulp#89acc5c17e115ec03ec6b17341c3ffdf5e2db837",
```

Reply:
Good point @benjifisher, although I don't think referencing a commit hash is a good idea as that will put is in the same place as before. It looks like there is more support, as well as being recommended in gulp's `README.md`, for using `gulpjs/gulp#v4.0.0-alpha.3. So I'm going to try swapping out the `^4.0.0` for `gulpjs/gulp#v4.0.0-alpha.3`.